public/pages/CreateDetector/components/ConfigureFieldMapping/containers/ConfigureFieldMapping.tsx

```diff
@@ -143,12 +143,23 @@ export default class ConfigureFieldMapping extends Component<
     return (
       <div>
         {!isEdit && (
-          <div>
+          <>
             <EuiTitle size={'l'}>
               <h3>{createDetectorSteps[DetectorCreationStep.CONFIGURE_FIELD_MAPPING].title}</h3>
             </EuiTitle>
+
+            <EuiText>
+              <p>
+                <EuiTextColor color="subdued">
```

Comment:
Should this be extracted to it's own component and used everywhere?

Reply:
`EuiText` has a `color` prop which can be used instead of wrapping the text in `EuiTextColor` and `p`.